src/index.js

```diff
@@ -500,6 +504,43 @@ const indy = {
     return [credDefId, JSON.parse(credDef)]
   },
 
+  async buildGetRevocRegDefRequest(submitterDid: Did | null, revocRegDefId: string): Promise<LedgerRequest> {
+    if (Platform.OS === 'ios') {
+      throw new Error(`Unsupported operation! Platform: ${Platform.OS}`)
+    }
+    return JSON.parse(await IndySdk.buildGetRevocRegDefRequest(submitterDid, revocRegDefId))
+  },
+
+  async parseGetRevocRegDefResponse(getRevocRegResponse: LedgerRequestResult): Promise<GetRevocRegDefResponse> {
+    if (Platform.OS === 'ios') {
+      throw new Error(`Unsupported operation! Platform: ${Platform.OS}`)
+    }
+    const [revocRegDefId, revocRegDef] = await IndySdk.parseGetRevocRegDefResponse(JSON.stringify(getRevocRegResponse))
+    return [revocRegDefId, JSON.parse(revocRegDef)]
+  },
+
+  async buildGetRevocRegDeltaRequest(
+    submitterDid: Did | null,
+    revocRegDefId: string,
+    from: number = 0,
+    to: number = new Date().getTime()
+  ): Promise<LedgerRequest> {
+    if (Platform.OS === 'ios') {
+      throw new Error(`Unsupported operation! Platform: ${Platform.OS}`)
+    }
+    return JSON.parse(await IndySdk.buildGetRevocRegDeltaRequest(submitterDid, revocRegDefId, from, to))
+  },
+
+  async parseGetRevocRegDeltaResponse(getRevocRegDeltaResponse: string): Promise<[string, object]> {
```

Comment:
Return data type cointais just `[string, object]` but it seems there is another parameter `timestamp`
```suggestion
  async parseGetRevocRegDeltaResponse(getRevocRegDeltaResponse: string): Promise<[string, object, number]> {
```